key/src/address.rs

```diff
@@ -33,6 +35,25 @@ pub struct FullAddress {
     pub address: Address,
 }
 
+impl FullAddress {
```

Comment:
`fmt::Display` implementation for `FullAddress` looks almost the same. What's the difference here?